charts/gateway-helm/templates/envoy-proxy-topology-injector-webhook.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.topologyInjector.enabled }}
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: 'envoy-gateway-topology-injector.{{ .Release.Namespace }}'
```

Comment:
imo since its unique to a ns, we dont need to suffix the ns

Reply:
oh nvm its cluster scoped

Reply:
This is a cluster scoped resource though. The actions it takes is scoped to an individual namespace but there would be conflicts between namespaced deployments otherwise.